tests/setup_services.py

```diff
@@ -327,10 +327,14 @@ def stop():
 
 
 async def setup_timelord(
-    port, full_node_port, rpc_port, vdf_port, sanitizer, consensus_constants: ConsensusConstants, b_tools: BlockTools
+    full_node_port,
+    sanitizer,
+    consensus_constants: ConsensusConstants,
+    b_tools: BlockTools,
+    vdf_port: uint16 = uint16(0),
+    rpc_port: uint16 = uint16(0),
```

Comment:
maybe we could even remove these port parameters here and above. Any service where we can set the port to 0, we should just hard code that in the tests.

Reply:
hmm but in the test we would then have to write to the config, which seems like a weird thing to do in the test no?

Reply:
I managed to remove a lot of them